<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormKorr</class>
 <widget class="QWidget" name="FormKorr">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Состояние корректоров</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Состояние на</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="DateEdit" name="dateEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonUpdCurrentTime">
          <property name="text">
           <string>Обновить на текущее время</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSplitter" name="splitterStat">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="DbViewer" name="tableViewStat">
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="labelStatGlass">
            <property name="text">
             <string>Стекло</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="DbViewer" name="tableViewStatGlass">
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelStatPar">
            <property name="text">
             <string>Выходные параметры стекла</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="DbViewer" name="tableViewStatPar"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Загрузка корректоров</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>С</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="DateEdit" name="dateEditBeg"/>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>По</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="DateEdit" name="dateEditEnd"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonUpd">
          <property name="text">
           <string>Обновить</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSplitter" name="splitterLoad">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="DbViewer" name="tableViewLoad">
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="labelLoadGlass">
            <property name="text">
             <string>Загрузка</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="DbViewer" name="tableViewLoadGlass"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DateEdit</class>
   <extends>QDateEdit</extends>
   <header>db/dbviewer.h</header>
  </customwidget>
  <customwidget>
   <class>DbViewer</class>
   <extends>QTableView</extends>
   <header>db/dbviewer.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
